engine/lib/views.php

```diff
@@ -1604,6 +1604,8 @@ function elgg_views_boot() {
 	$elgg_js_url = elgg_get_simplecache_url('js', 'elgg');
 	elgg_register_js('elgg', $elgg_js_url, 'head');
 
+	elgg_extend_view('js/elgg', 'js/elgg/spinner.js', 501);
```

Comment:
Prefer not to extend this view any more if possible. I know it creates more http requests but this is not something that should block page rendering anyways so I think that's OK.